### PR TITLE
Task revision/alert feauture

### PR DESCRIPTION
**Pull request by MsweliP**
Implements Tank Graph and  Tank Component.

### CODE REVIEW

File: waterlog/src/App.scss [hunk 1]
```diff
@@ -85,6 +85,21 @@ hr {
 	padding-top: 37px; 
 }
 
+.tank-graph{
+	background-color: white;
+    margin: 20px;
```
Comment (FredJEhlers) [inline waterlog/src/App.scss:90]: inconsistent indentations

File: waterlog/src/Components/DailyWastage.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { Line, defaults } from 'react-chartjs-2';
 import { Globals } from './../Globals';
 
 const DailyWastageComponent = (props) => {
+  console.log(props);
```
Comment (FredJEhlers) [inline waterlog/src/Components/DailyWastage.js]: remove console logs

Comment (FredJEhlers) [reply waterlog/src/Components/DailyWastage.js]: in all files

File: waterlog/src/Components/TankComponent.js [hunk 3]
```diff
@@ -4,39 +4,41 @@ import { connect } from 'react-redux';
...
-		} else if (percent >= 41 && percent <= 79) {
+		} else if (level.percentage >= 41 && level.percentage <= 79) {
 			return (
 				<div>
-					<img src={images.tank_orange} className="image" alt="50% tank" />
-					<p id="percentage">{percent}%</p>
+					<img src={images.tank_orange} className="image" alt="tank" />
+					<p id="percentage">{level.percentage}%</p>
+					<p>{level.levelStatus}</p>
+				    <p>{level.instruction}</p>
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankComponent.js]: inconsistent indentations

File: waterlog/src/Components/TankGraph.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {Line, defaults} from 'react-chartjs-2';
+
+const TankGraph = () => {
+const data = {
+  labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+  datasets: [
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankGraph.js]: inconsistent indentation vs ` waterlog/src/Components/TankComponent.js`

File: waterlog/src/Components/Wastage.js [hunk 5]
```diff
@@ -36,6 +39,14 @@ class Wastage extends Component {
 	};
 
 	render() {
+		const { dailyError, dailyLoading, dailyWaste,monthlyError,monthlyWaste,monthlyLoading,seasonalError,seasonalLoading,seasonWaste } = this.props;
```
Comment (FredJEhlers) [inline waterlog/src/Components/Wastage.js]: split over multiple lines

File: waterlog/src/Stylesheets/_tank.scss [hunk 6]
```diff
@@ -29,5 +29,28 @@ h6{
...
+    font-size: 10pt;
+}
+
+#homepage_tank_percentage{
+    position: relative;
+    display: inline-block;
+    top: -60px;
+    color: #FFF;
+    font-size: 10pt;
+}
```
Comment (FredJEhlers) [inline waterlog/src/Stylesheets/_tank.scss:54]: empty line at eof

Comment (FredJEhlers) [reply waterlog/src/Stylesheets/_tank.scss:54]: ensure this across all files

File: waterlog/src/actions/Types_BACKUP_15920.js [hunk 7]
```diff
@@ -0,0 +1,75 @@
+export const FETCH_SENSORS = 'FETCH_SENSORS';
```
Comment (FredJEhlers) [inline waterlog/src/actions/Types_BACKUP_15920.js]: why you committing this file

File: waterlog/src/actions/Types_BASE_15920.js [hunk 8]
```diff
@@ -0,0 +1,62 @@
+export const FETCH_SENSORS = 'FETCH_SENSORS';
```
Comment (FredJEhlers) [inline waterlog/src/actions/Types_BASE_15920.js]: why are you committing this file

File: waterlog/src/actions/Types_LOCAL_15920.js [hunk 9]
```diff
@@ -0,0 +1,74 @@
+export const FETCH_SENSORS = 'FETCH_SENSORS';
```
Comment (FredJEhlers) [inline waterlog/src/actions/Types_LOCAL_15920.js]: why are you committing this file

File: waterlog/src/actions/Types_LOCAL_15920.js [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+export const FETCH_SEGMENTS_RESOLVED_BEGIN = 'FETCH_SEGMENTS_RESOLVED_BEGIN';
+export const FETCH_SEGMENTS_RESOLVED_SUCCESS = 'FETCH_SEGMENTS_RESOLVED_SUCCESS';
+export const FETCH_SEGMENTS_RESOLVED_FAILURE = 'FETCH_SEGMENTS_RESOLVED_FAILURE'; 
+//Events actions
+export const FETCH_SEGMENT_EVENTS_BEGIN = 'FETCH_SEGMENT_EVENTS_BEGIN';
+export const FETCH_SEGMENT_EVENTS_SUCCESS = 'FETCH_SEGMENT_EVENTS_SUCCESS';
+export const FETCH_WASTAGE_DAILY = "FETCH_WASTAGE_DAILY";
+export const FETCH_WASTAGE_MONTHLY = "FETCH_WASTAGE_MONTHLY";
+export const FETCH_WASTAGE_SEASONALLY = "FETCH_WASTAGE_SEASONALLY";
+//export const FETCH_WASTAGE_MONTHLY_BEGIN = "FETCH_WASTAGE_MONTHLY_BEGIN";
```
Comment (FredJEhlers) [inline waterlog/src/actions/Types_LOCAL_15920.js]: gc

File: waterlog/src/actions/Types_REMOTE_15920.js [hunk 11]
```diff
@@ -0,0 +1,72 @@
+export const FETCH_SENSORS = 'FETCH_SENSORS';
```
Comment (FredJEhlers) [inline waterlog/src/actions/Types_REMOTE_15920.js]: why are you committing this

File: waterlog/src/actions/WastageDaily.js [hunk 12]
```diff
@@ -1,18 +1,38 @@
-import { FETCH_WASTAGE_DAILY, FETCH_WASTAGE_MONTHLY , FETCH_WASTAGE_SEASONALLY} from "./Types";
+import { FETCH_WASTAGE_DAILY_BEGIN, FETCH_WASTAGE_DAILY_FAILURE,FETCH_WASTAGE_DAILY_SUCCESS,handleErrors} from "./Types";
+
+//import { FETCH_WASTAGE_DAILY, FETCH_WASTAGE_MONTHLY , FETCH_WASTAGE_SEASONALLY, FETCH_WASTAGE_DAILY_BEGIN, FETCH_WASTAGE_DAILY_FAILURE, FETCH_WASTAGE_DAILY_SUCCESS} from "./Types";
```
Comment (FredJEhlers) [inline waterlog/src/actions/WastageDaily.js]: gc

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/src/Components/DailyWastage.js [hunk 13]
```diff
@@ -50,18 +49,18 @@ const DailyWastageComponent = (props) => {
...
   }
-    defaults.global.legend.display = false;
-    return (
-      <div className="wastage-graph">
+  defaults.global.legend.display = false;
+  return (
+    <div className="wastage-graph">
 
-      <div className="head"><b>R {(Math.round(sum*Globals.WATER_COST))}</b><b className="dailysubhead"> lost so far</b></div>
+      <div className="head"><b>R {(Math.round(sum * Globals.WATER_COST))}</b><b className="dailysubhead"> lost so far</b></div>
```
Comment (FredJEhlers) [inline waterlog/src/Components/DailyWastage.js]: Inconsistent indentation

File: waterlog/src/actions/WastageMonthly.js [hunk 14]
```diff
@@ -0,0 +1,32 @@
+import { FETCH_WASTAGE_MONTHLY_BEGIN, FETCH_WASTAGE_MONTHLY_FAILURE,FETCH_WASTAGE_MONTHLY_SUCCESS,handleErrors} from "./Types";
+
+//import { FETCH_WASTAGE_DAILY, FETCH_WASTAGE_MONTHLY , FETCH_WASTAGE_SEASONALLY, FETCH_WASTAGE_DAILY_BEGIN, FETCH_WASTAGE_DAILY_FAILURE, FETCH_WASTAGE_DAILY_SUCCESS} from "./Types";
```
Comment (FredJEhlers) [inline waterlog/src/actions/WastageMonthly.js]: GC

File: waterlog/src/actions/WastageSeasonally.js [hunk 15]
```diff
@@ -0,0 +1,32 @@
+import { FETCH_WASTAGE_SEASONALLY_BEGIN, FETCH_WASTAGE_SEASONALLY_FAILURE,FETCH_WASTAGE_SEASONALLY_SUCCESS,handleErrors} from "./Types";
+
+//import { FETCH_WASTAGE_DAILY, FETCH_WASTAGE_MONTHLY , FETCH_WASTAGE_SEASONALLY, FETCH_WASTAGE_DAILY_BEGIN, FETCH_WASTAGE_DAILY_FAILURE, FETCH_WASTAGE_DAILY_SUCCESS} from "./Types";
```
Comment (FredJEhlers) [inline waterlog/src/actions/WastageSeasonally.js]: GC

File: waterlog/src/reducers/WastageGraphMonthlyReducer.js [hunk 16]
```diff
@@ -1,3 +1,4 @@
+import { FETCH_WASTAGE_MONTHLY_BEGIN,FETCH_WASTAGE_MONTHLY_SUCCESS,FETCH_WASTAGE_MONTHLY_FAILURE } from '../actions/Types';
```
Comment (FredJEhlers) [inline waterlog/src/reducers/WastageGraphMonthlyReducer.js]: Split over multiple lines

File: waterlog/src/reducers/WastageGraphSeasonallyReducer.js [hunk 17]
```diff
@@ -1,4 +1,4 @@
-import { FETCH_WASTAGE_SEASONALLY } from '../actions/Types';
+import { FETCH_WASTAGE_SEASONALLY_BEGIN,FETCH_WASTAGE_SEASONALLY_SUCCESS,FETCH_WASTAGE_SEASONALLY_FAILURE } from '../actions/Types';
```
Comment (FredJEhlers) [inline waterlog/src/reducers/WastageGraphSeasonallyReducer.js:1]: Split over multiple lines

Comment (MsweliP) [reply waterlog/src/reducers/WastageGraphSeasonallyReducer.js:1]: Done

File: waterlog/src/reducers/WastageGraphMonthlyReducer.js (hunk 16 above)
Comment (MsweliP) [reply waterlog/src/reducers/WastageGraphMonthlyReducer.js]: Done

File: waterlog/src/actions/WastageSeasonally.js (hunk 15 above)
Comment (MsweliP) [reply waterlog/src/actions/WastageSeasonally.js]: Done

File: waterlog/src/actions/WastageMonthly.js (hunk 14 above)
Comment (MsweliP) [reply waterlog/src/actions/WastageMonthly.js]: Done

File: waterlog/src/Components/DailyWastage.js (hunk 13 above)
Comment (MsweliP) [reply waterlog/src/Components/DailyWastage.js]: Done

File: waterlog/src/Components/TankComponent.js [hunk 18]
```diff
@@ -13,9 +13,22 @@ export class TankComponent extends Component {
 	componentWillMount() {
 		this.props.fetchTankLevelById(this.props.id);
 	}
+<<<<<<< HEAD
```
Comment (FredJEhlers) [inline waterlog/src/Components/TankComponent.js]: You need to clean up the code here from your merge conflicts

File: waterlog/src/actions/Types.js [hunk 19]
```diff
@@ -19,6 +19,7 @@ export const FETCH_SEGMENTS_RESOLVED_FAILURE = 'FETCH_SEGMENTS_RESOLVED_FAILURE'
 //Events actions
 export const FETCH_SEGMENT_EVENTS_BEGIN = 'FETCH_SEGMENT_EVENTS_BEGIN';
 export const FETCH_SEGMENT_EVENTS_SUCCESS = 'FETCH_SEGMENT_EVENTS_SUCCESS';
+<<<<<<< HEAD
```
Comment (FredJEhlers) [inline waterlog/src/actions/Types.js]: You need to clean up the code here from your merge conflicts

File: waterlog/src/actions/WastageDaily.js [hunk 20]
```diff
@@ -19,6 +19,7 @@ export const fetchWastageDailyFailure = (error) => ({
 
 export const fetchWastageDaily = () => (dispatch) => {
   dispatch(fetchWastageDailyBegin())
+<<<<<<< HEAD
```
Comment (FredJEhlers) [inline waterlog/src/actions/WastageDaily.js]: You need to clean up the code here from your merge conflicts

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**